mgmt/rest/v2/metric.go

```diff
@@ -0,0 +1,92 @@
+package v2
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"sort"
+
+	"github.com/codegangsta/negroni"
+	"github.com/intelsdi-x/snap/core"
+	"github.com/intelsdi-x/snap/mgmt/rest/rbody"
+	"github.com/intelsdi-x/snap/mgmt/rest/rbody/v2"
+)
+
+func RespondWithMetrics(host string, mets []core.CatalogedMetric, w http.ResponseWriter) {
```

Comment:
Just thought it would make sense to use same variable name. I didn't see "mets" before, AFAIK "metrics" or "mts" is more common, with the singular "metric" or "m".